### PR TITLE
i18n(es): Fix typo in `server-side-rendering.mdx` & `4-layouts/2.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Fix typo in `server-side-rendering.mdx` & `4-layouts/2.mdx`

### CODE REVIEW

File: src/content/docs/es/tutorial/4-layouts/2.mdx [hunk 1]
```diff
@@ -175,8 +175,8 @@ const { frontmatter } = Astro.props;
 
 ### Recursos
 
-- [Layouts de Markdown en Astro](/es/guides/markdown-content/#layout-en-el-frontmatter)
+- [Diseños de Markdown en Astro](/es/guides/markdown-content/#layout-en-el-frontmatter)
```
Comment (dreyfus92) [inline src/content/docs/es/tutorial/4-layouts/2.mdx:178]:
```suggestion
- [Plantillas de Markdown en Astro](/es/guides/markdown-content/#layout-en-el-frontmatter)
```

File: src/content/docs/es/tutorial/4-layouts/2.mdx [hunk 2]
```diff
@@ -175,8 +175,8 @@ const { frontmatter } = Astro.props;
...
 
-- [Markdown Layout Props](/es/core-concepts/layouts/#props-de-plantillas-markdown)
+- [Props de diseño para Markdown.](/es/core-concepts/layouts/#props-de-plantillas-markdown)
```
Comment (dreyfus92) [inline src/content/docs/es/tutorial/4-layouts/2.mdx:180]:
```suggestion
- [Props de plantilla para Markdown](/es/core-concepts/layouts/#props-de-plantillas-markdown)
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59 😁 te dejo algunas sugerencias.

**Comment by Waxer59** [comment]
Listo 🤠, gracias por la revision!

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**